manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml

```diff
@@ -36,7 +36,6 @@ metadata:
                   "nodeCount": 3,
                   "redundancyPolicy": "SingleRedundancy",
                   "storage": {
-                    "storageClassName": "gp2",
```

Comment:
in this case the `"size": "200G"` does nothing when the elasticsearch operator ends up resolving this... so by default you'll end up with ephemeral storage. There isn't an easy way to then move any captured log records into a cluster backed with storage. So i'm not sure which is the lesser evil here... loss of logs or needing to manually remove a pvc once the correctly backed ones are created...

Reply:
Oh, the fall back is ephemeral, rather than the default storage class? Hmm

Reply:
> in this case you'll end up with ephemeral storage. 

It will not be ephemeral as  the cluster has a default class so you will get whatever that is.  The fact you define a size will beget you a PVC

Reply:
Note this is dependent also on a forth coming change in the EO